src/pages/data/download.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import ContentfulContent from '~components/common/contentful-content'
+import Panel from '~components/pages/data/download/panel'
+import Fields from '~components/pages/data/download/fields'
+import DownloadLink from '~components/pages/data/download/download-link'
+import Layout from '~components/layout'
+
+export default ({ data }) => (
+  <Layout
+    title="Data Download"
+    path="/data/download"
+    navigation={data.contentfulNavigationGroup.pages}
+  >
+    <ContentfulContent
+      content={
+        data.contentfulSnippet.childContentfulSnippetContentTextNode
+          .childMarkdownRemark.html
+      }
+      id={data.contentfulSnippet.contentful_id}
+    />
+
+    <h2>US Data</h2>
+    <DownloadLink
+      desc="Download all available US data"
+      path="/api/v1/us/daily.csv"
+    />
+    <Panel label="Field Descriptions">
+      <Fields schema="Us" />
```

Comment:
@kevee Is it possible that the name of the schema could change? If so I could get the schema name from the path, e.g.
```
import definition from '../../../_api/v1/openapi.json'
...
  <Fields
    schema={
      definition.paths['/api/v1/states/daily.{format}'].get.responses[200].content['text/csv']
        .schema.items.$ref.split('/').pop()
    }
  />
...
```